config/opal_setup_wrappers.m4

```diff
@@ -291,9 +293,11 @@ AC_DEFUN([RPATHIFY_LDFLAGS_INTERNAL],[
                esac
            done
 
-           # Now add in the RPATH args for @{libdir}, and the RUNPATH args
-           rpath_tmp=`echo ${$2} | sed -e s/LIBDIR/@{libdir}/`
-           $1="${$1} $rpath_out $rpath_tmp ${$3}"
+           # add in the RPATH args for @{libdir}, and the RUNPATH
+           # args.  The install libdir goes first, so that we prefer
+           # our libmpi over any imposter libmpi we might find.
+           rpath_install_libdir=`echo ${$2} | sed -e s/LIBDIR/@{libdir}/`
```

Comment:
Can you add `rpath_install_libdir` to the `OPAL_VAR_SCOPE_PUSH`, above?

Reply:
yeah, there was no need for the rename.  fixed.